### DISCUSSION TITLE
Sayı dizisi sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#292] [Zeka Sorusu]
8, 7, 7, 5, 10, 7, 21, 17, ?

Soru işaretli yere ne gelecektir

**Comment by yigidi** [comment]
Bu dizideki sayıların ardışık farklarına bakalım:

7 - 8 = -1
7 - 7 = 0
5 - 7 = -2
10 - 5 = 5
7 - 10 = -3
21 - 7 = 14
17 - 21 = -4

Ardışık farklara baktığımızda, -1, 0, -2, 5, -3, 14, -4 şeklinde bir desen görüyoruz. Bu desene göre, bir sonraki ardışık farkın 14'den 4 daha az olması beklenir, yani 14 - 4 = 10 olur.

Şimdi, son bulduğumuz ardışık farkı son verilen sayıya ekleyelim:

17 + 10 = 27

Sonuç olarak, soru işaretli yere 27 gelmelidir.
